### PR TITLE
E2e to sync 2 dev nodes

### LINKED ISSUE #812: Lipp2p: need to close resources after use
When I work on #726, I used a `beacon` node to connect to `dev` node and start/stop them, then the test cannot exit so I have to use `--exit` flag in package.json

Since we should not connect `beacon` node to `dev` node thru shared state, I don't have this issue anymore but it's worth to investigate.

To reproduce, write an e2e test:
+ Start a dev node with 8 validators
+ Start another dev node without validator
+ Wait for blocks to be received on the 2nd node thru initial/regular sync
+ Make sure we can start/stop/sync all nodes smoothly in the test

### PR DESCRIPTION
**Pull request by twoeths**
# Goal
+ resolve #812

### CODE REVIEW

**Comment by twoeths** [comment]
Due to #864 , the emitted events are all `inbound` instead of `inbound` & `outbound` hence no handshake happened and dev node 2 can't sync to dev node 1.

**Comment by twoeths** [comment]
It works now as I connect 2 dev nodes manually

File: packages/lodestar/src/opPool/modules/attestation.ts [hunk 1]
```diff
@@ -18,7 +19,7 @@ export class AttestationOperations extends OperationsModule<Attestation> {
       return attestation.data.index === committeeIndex
           && computeEpochAtSlot(this.config, attestation.data.slot) === epoch
           //filter out aggregated attestations
-          && attestation.aggregationBits.length === 1;
+          && isUnaggregatedAttestation(attestation);
```
Comment (twoeths) [inline packages/lodestar/src/opPool/modules/attestation.ts:22]: need confirmation

Comment (wemeetagain) [reply packages/lodestar/src/opPool/modules/attestation.ts:22]: :+1:

**Review by wemeetagain — APPROVED** [review]

**Pull request merged**